.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@meshack-mbuvi remove this file from history.

Reply:
@parseendavid Your request is completed.